include/lsst/afw/detection/FootprintSet.h

```diff
@@ -119,14 +151,41 @@ class FootprintSet : public lsst::daf::base::Citizen {
 
 #endif
 
+    /**
+     * Construct an empty FootprintSet given a region that its footprints would have lived in
+     *
+     * @param region the desired region
+     */
     FootprintSet(geom::Box2I region);
+    /**
+     * Copy constructor
+     *
+     * @param rhs the input FootprintSet
+     */
     FootprintSet(FootprintSet const&);
     FootprintSet(FootprintSet const& set, int rGrow, FootprintControl const& ctrl);
```

Comment:
Not sure I understand this comment. Doxygen reorders class members anyway, so order matters only on the summary list; in addition, detailed docs are displayed for only one overload at a time. So I'm not sure what this change would accomplish.